src/target/riscv/riscv.c

```diff
@@ -477,203 +575,205 @@ static int maybe_add_trigger_t1(struct target *target,
 	const uint32_t bpcontrol_bpmatch = 0xf << 7;
 	const uint32_t bpcontrol_bpaction = 0xff << 11;
 
+	ret = find_trigger(target, 1, 1, &idx);
```

Comment:
Everywhere that find_trigger() is called, please use the macro name for the type, e.g. CSR_TDATA1_TYPE_LEGACY in this instance.